### DISCUSSION TITLE
Addition of Asserts for special data types XINT, UXINT, XWORD?

### DISCUSSION THREAD

**Discussion by AndyP01** [tcunit/TcUnit#164] [Ideas]
Could it be beneficial to add asserts for the special data types XINT, UXINT and XWORD?

See here:
https://infosys.beckhoff.com/english.php?content=../content/1033/tc3_plc_intro/2529448075.html&id=

**Comment by sagatowski** [comment]
I think it's a good idea. I'll create an issue from this.

**Comment by sagatowski** [comment]
Issue created. See https://github.com/tcunit/TcUnit/issues/165.

**Comment by sagatowski** [comment]
Fixed links :-)

**Comment by sagatowski** [comment]
@AndyP01 If you have time, you are of course welcome to create a PR for this!
